excel_graph_parser/graph_parser.py

```diff
@@ -248,8 +260,8 @@ def get_figure_titles(self):
         return figure_list
 
     @staticmethod
-    def create_ploty_figure(chart_data: dict):
-        """Creates ploty figure based on the extracted chart data"""
+    def _create_plotly_figure(chart_data: dict) -> dict:

```

Comment:
Note that this breaks compatibility if actually used in apps, which I highly doubt